go/vt/vterrors/vterrors.go

```diff
@@ -327,5 +327,8 @@ func Equals(a, b error) bool {
 // Print is meant to print the vtError object in test failures.
 // For comparing two vterrors, use Equals() instead.
 func Print(err error) string {
-	return fmt.Sprintf("%v: %v\n", Code(err), err.Error())
+	if err != nil {
+		return fmt.Sprintf("%v: %v\n", Code(err), err.Error())
+	}
+	return fmt.Sprintf("%v: nil\n", Code(err))
```

Comment:
if error is nil, what does this Code(err) mean?

Reply:
Code(err) returns a Code_OK on nil.